docs/PullPayments.md

```diff
@@ -0,0 +1,123 @@
+# Pull payments
+
+## Introduction
+
+Traditionally, to make a Bitcoin payment, a receiver shares their bitcoin address and the sender later sends money to this address.
+Such system is called `Push payment` as the sender initiates the payment while the receiver may be unavailable, in effect `pushing` the payment to the receiver.
+
+However, what about reversing the role?
+What if, instead of a sender `pushing` the payment, the sender allows the receiver to `pull` the payment at a time the receiver seems fit?
+This is the concept of `Pull payment`. This allows several new applications such as:
+
+* A subscription service (where the subscriber allows the service to pull money every month)
+* Refunds (where the merchant allows the customer to pull the refund money to his wallet when it seems fit)
+* Time based billing freelancing (where the person hiring allows the freelance to pull money to his wallet as time get reported)
+* Patronage (where the patron allows the recipient to pull money every month to continue supporting his work)
+* Automatic selling (where a customer of an exchange would allow an exchange to pull money from his wallet to automatically sell every month)
+* Balance withdraw system (where a high volume service allows users to request withdrawals from their balance, the service can then easily batch all the payouts to many users, at fixed intervals)
+
+## Concept
+
+When a sender configures a `Pull payment`, they can configure a number of properties:
+* Start date
+* End date (optional)
+* A period (optional)
+* A limit amount
+* A unit (such as USD, BTC, Hours)
+* Available payment methods
+
+After this, the sender can share the pull payment with the receiver, allowing the receiver to `create a payout`.
+The receiver will choose for their payout:
+* Which payment method to use
+* Where to send the money
+
+Once a payout is created, it will count toward the `pull payment's limit` for the current `period`.
+The sender will then approve the payout by setting the `rate` in which the payout will be sent, and proceed to payment.
+
+For the sender, we provide an easy to use way to batch the payment of several payouts from the wallet of their BTCPay Server instance.
```

Comment:
> the wallet of their BTCPay Server instance.

In docs, we tend to refer to it as an internal BTCPay Server wallet, so I suggest you use that for consistency and also link to Wallet.md